### PR TITLE
Apply style banner section

### PR DESCRIPTION
**Pull request by Yukit02**
### Implementation details

- Subject says it All

### CODE REVIEW

File: src/templates/Footer.tsx [hunk 1]
```diff
@@ -5,31 +5,31 @@ import Link from 'next/link';
...
+import { AppInfo } from '../utils/AppInfo';
 
 const Footer = () => (
   <Background color="bg-primary-900">
     <Section yPadding="p-8">
       <CenteredFooter
         iconList={(
           <>
-            <Link href={AppLink.twitter}>
+            <Link href={AppInfo.link.twitter}>
```
Comment (hoonsubin) [inline src/templates/Footer.tsx:16]: Please remove the `Link` element and use the `<a>` tag for external links

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/templates/Banner.tsx [hunk 2]
```diff
@@ -1,24 +1,57 @@
...
+              <path d="M23.954 4.569a10 10 0 01-2.825.775 4.958 4.958 0 002.163-2.723c-.951.555-2.005.959-3.127 1.184a4.92 4.92 0 00-8.384 4.482C7.691 8.094 4.066 6.13 1.64 3.161a4.822 4.822 0 00-.666 2.475c0 1.71.87 3.213 2.188 4.096a4.904 4.904 0 01-2.228-.616v.061a4.923 4.923 0 003.946 4.827 4.996 4.996 0 01-2.212.085 4.937 4.937 0 004.604 3.417 9.868 9.868 0 01-6.102 2.105c-.39 0-.779-.023-1.17-.067a13.995 13.995 0 007.557 2.209c9.054 0 13.999-7.496 13.999-13.986 0-.209 0-.42-.015-.63a9.936 9.936 0 002.46-2.548l-.047-.02z" />
+            </svg>
+          </SnsButton>
+
+          <SnsButton
+            btnColor="bg-discord"
+            appLink={AppInfo.link.discord}
+            appName={AppInfo.name.discord}
+          >
+            <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (Yukit02) [reply src/templates/Banner.tsx:47]: Plz tell me How to use props with SVG.
And if I use svg in img src can not change color with only special way.

[the remark this answers is not in the document]
Comment (hoonsubin) [reply src/templates/Banner.tsx:47]: how about creating it as a separate component that takes the color as one of its props?

**Comment by Yukit02** [comment]
I fixed all.
plz check again.

**Review by hoonsubin — APPROVED** [review]
LGTM

**Pull request merged**